### ISSUE TITLE
Ferdi doesn't launch after update

### ISSUE THREAD

**Issue by keunes** [getferdi/ferdi#701]
**Describe the bug**
After installing updates on my computer just now, Ferdi doesn't launch - or rather: is stuck on the purple screen (before even log-in shows).

A few seconds I see a white-ish flash on the screen, relaunching a few times it seems that that flash shows some field-areas (but in white, only).

![Schermafdruk van 2020-05-01 17-50-33](https://user-images.githubusercontent.com/11229646/80819064-95c0f580-8bd4-11ea-9858-650e0578df56.png)
![Schermafdruk van 2020-05-01 21-57-02](https://user-images.githubusercontent.com/11229646/80837265-c580f500-8bf6-11ea-9630-102729bf9284.png)

**Desktop:**
- OS: Linux
- Ferdi Version - I believe 5.5.0 now (before I believe 5.4.3). Installed via __flatpak__
- Server Used: none (local)

**Comment by kytwb** [comment]
Thank you for the report @keunes. Can you confirm you were using Ferdi without an account? Sorry for the bad experience with the update, this has been reported in https://github.com/getferdi/ferdi/issues/543 as well. We had to make trade-offs as new users were still geting 5.4.3 via the website and other stable release channels, where a lot of things wasn't working anymore 🙏 Let's continue the discussion in the other thread? We will do better with the next iteration. Thank you for your patience!

**Comment by jeannsensei** [comment]
Hi, this happened to me too.
I noticed that in the previous dev version 5.5.0-gm.2, Ferdi breaks when I set up a password lock.
Then I had this problem on Windows 10,  on my Ubuntu system I did not update because of this.

The solution that I did was to delete everything using geek uninstaller.
Then, I installed the 5.4.3 version and it prompted me to update directly to the 5.5.0.
This resolved the issue BUT if I enable password lock, the app just crashes and this happens again.
